src/config/config.ts

```diff
@@ -0,0 +1,4 @@
+import axios from 'axios';
```

Comment:
Commenting here but file should be in `/axios` folder named `index.ts` or `axios.ts`